scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+open_browser_and_login() {
+    echo "Logging you in via $1 ..."
+    platform="$(uname)"
+    if [[ "$platform" == "Linux" ]]; then
+        xdg-open "$1" >/dev/null &
+    elif [[ "$platform" == "Darwin" ]]; then
+        open "$1" &
+    else
+        echo "Unsupported platform '$platform', please open $1 in a browser" 1>&2
+    fi
+}
+
+login_if_no_auth_error() {
+    auth_error="$(sed -E '/^.*#error=([^&]+)(&.*)?$/!d;s//\1/' <<<"$target_url" | tr '+' ' ')"
+    if [[ -z "$auth_error" ]]; then
+        open_browser_and_login $1
+    else
+        echo "Authentication error: ${auth_error}" 1>&2
+    fi
+}
+
+login_stackrox() {
+_output="$(mktemp)"
+if curl -sSkf -u "admin:${STACKROX_ADMIN_PASSWORD}" --output "${_output}" -w '%{redirect_url}' "https://localhost:8000/sso/providers/basic/4df1b98c-24ed-4073-a9ad-356aec6bb62d/challenge?micro_ts=0"; then
+    target_url="$(cat "${_output}")"
+    if [[ -n "$target_url" ]]; then
+        login_if_no_auth_error target_url
```

Comment:
Should this be
```suggestion
        login_if_no_auth_error "$target_url"
```
?

Reply:
Yes you're right. I've already changed the functions to one as mentioned above so this is now redundant.